README.md

```diff
@@ -779,14 +779,14 @@ The `LoadState` type is used in several of the metric [attribution objects](#att
  * State descriptions:
  * - `loading`: the initial document response has not yet been fully downloaded
  *   and parsed. This is equivalent to the corresponding `readyState` value.
- * - `domInteractive`: the document has been fully loaded and parsed, but
+ * - `interactive`: the document has been fully loaded and parsed, but
  *   scripts may not have yet finished loading and executing.
- * - `domContentLoaded`: the document is fully loaded and parsed, and all
+ * - `content-loaded`: the document is fully loaded and parsed, and all
```

Comment:
This one seems an odd change to me. `domContentLoaded` is a well-known [event](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) and it uses capitalisation we've used before: https://web.dev/critical-rendering-path-analyzing-crp/

Reply:
> `domContentLoaded` is a well-known [event](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)

The event is actually `DOMContentLoaded` :) but either way, note that this API is referring to a state (the timespan between `domContentLoadedEventStart` and `domComplete` in the [Navigation Timing API](https://w3c.github.io/navigation-timing/#sec-PerformanceNavigationTiming)) and not the DOM event itself, so I don't think it's important to be consistent with the event name.

Also the Design Principles doc [specifically mentions cases](https://www.w3.org/TR/design-principles/#naming-consistency:~:text=Note%20that%20in%20particular%2C%20when%20a%20HTML%20attribute%20is%20reflected%20as%20a%20property%2C%20the%20attribute%20and%20property%E2%80%99s%20casings%20won%E2%80%99t%20necessarily%20match.%20For%20example%2C%20the%20HTML%20attribute%20ismap%20on%20img%20elements%20is%20reflected%20as%20the%20isMap%20property%20on%20HTMLImageElement.) where the same term may have different casing based on how it's used, and you can see an example of that with the differences between `DOMContentLoaded` (the event name) and `domContentLoadedEventStart` (the Navigation Timing interface property name).

BTW, I did go back and forth as to whether these names should be `'dom-interactive'` and `'dom-content-loaded'` or `'interactive'` and `'content-loaded'` (and I'm still torn). Using the name `'interactive'` is good because it matches the the [`document.readyState`](https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState) value, but on the other hand it's not a 1-to-1 match with the "interactive" readyState value, so maybe that warrants a different name to help differentiate. WDYT?

Reply:
Yeah I recognised it wasn’t exactly the event itself (hence why I was ok with different capitalisation more inline with the Navigation timing API.

I still think mentioning DOM in here somewhere (`dom-content-loaded`?) is better as a description of what it is. Just `content-loaded` seems very generic. And maybe the use of hyphens will make it more obvious it’s a state rather than the event or timing?

And I’m similarly torn between matching readystate and that being a good thing, or liable for confusion. I dunno. Naming things is hard…

Reply:
I'd be fine with this:

```ts
type LoadState = 'loading' | 'dom-interactive' | 'dom-content-loaded' | 'complete';
```

So `loading` and `complete` are the same as their `readyState` versions, and `dom-interactive` and `dom-content-loaded` are the sub-phases of `readyState === 'interactive'`. Let me know what you think.

Reply:
Works for me.

Reply:
Done.